rllib/core/learner/learner.py

```diff
@@ -852,7 +852,12 @@ def set_state(self, state: Mapping[str, Any]) -> None:
         # having both can become confusing. Can we simplify this API requirement?
         self._check_is_built()
         # TODO: once we figure out the optimizer format, we can set/get the state
-        self._module.set_state(state.get("module_state", {}))
+        module_state = state["module_state"]
```

Comment:
I could probably check for the existence of these keys first then error out if necessary.